project/src/main/webapp/script.js

```diff
@@ -62,11 +121,131 @@ function createPlaceElement(place) {
 }
 
 /**
- * Displays the query form to the user and hides the results, so that the user can try again with a
- * different query.
+ * Displays the query form to the user and hides the results, so that the user can try to get new
+ * results.
  */
 function tryAgain() {
   document.getElementById('query-form').style.display = 'block';
   document.getElementById('results').style.display = 'none';
   document.getElementById('place').innerHTML = '';
 }
+
+function addSearchBoxToMap(map, input) {
+  // Create the search box and link it to the UI element.
+  const searchBox = new window.google.maps.places.SearchBox(input);
```

Comment:
What is the input? A location? Maybe give it a more describing name

Reply:
Good point, I made it clearer. Thanks!